gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixRetriggeringJobCallable.java

```diff
@@ -143,7 +144,7 @@ public HelixRetriggeringJobCallable(
     this.jobsMapping = jobsMapping;
     this.locks = locks;
     this.metricContext = metricContext;
-    eventSubmitter = new EventSubmitter.Builder(this.metricContext, "gobblin.runtime").build();
+    eventSubmitter = new EventSubmitter.Builder(this.metricContext, JobMetrics.NAMESPACE).build();
```

Comment:
Replaced all raw usages of "gobblin.runtime" with JobMetrics.NAMESPACE for consistency and readability across jobs